addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12833,24 +12843,24 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#24116"
-msgid "Default TV Service"
+msgid "Default TV service"
```

Comment:
done... and not squashing and not pushing is really hard for me to keep track of what I changed.